tox.ini

```diff
@@ -1,16 +1,47 @@
 [tox]
-envlist = py33,py34,py35,py36,coverage,flake8,isort
+envlist = {py33,py34,py35,py36,coverage}-{cext,nocext},flake8,isort
 
 [testenv]
+setenv =
+    cext: WEBSOCKETS_CEXT_BUILD=1
+    nocext: WEBSOCKETS_CEXT_BUILD=0
 deps =
     py33: asyncio
-commands = python -m unittest
+commands =
+    python setup.py clean --all
+    python -c "import glob, os; [os.remove(f) for files in [glob.glob('./websockets/' + ext) for ext in ['*.so', '*.pyd']] for f in files]"
```

Comment:
Now that I reworked to suppress the environment variables, the updated `tox.ini` has no need for external tools. It is however a bit complex to avoid duplication of code (the coverage environment is now multi-platform since coverage of the C code is "not possible" on Windows). 
I moved the `rm` part to `setup.py`which got a little bit longer.